quiz.ipynb

```diff
@@ -0,0 +1,89 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 26,
+   "id": "eb4efd5c",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "['myself dimpal', '']\n",
+      "['i am live in bardoli', '']\n",
+      "['my favourite color is black', '']\n",
+      "['my hobby is travelling', '']\n",
+      "['i am studying at vidyabharti college trust', '']\n",
+      "['my age is 18', '']\n",
+      "[]\n"
+     ]
+    }
+   ],
+   "source": [
+    "import csv\n",
+    "with open('C:/22bca191/python/intro1.txt','r',newline='') as read_obj:\n",
+    "    read=csv.reader(read_obj)\n",
+    "    for i in read:\n",
+    "        print(i)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 27,
+   "id": "4553f827",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "i am live in bardoli,\r\n",
+      "\n",
+      "my hobby is travelling,\r\n",
+      "\n",
+      "my age is 18,\r\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "import csv\n",
+    "with open('C:/22bca191/python/intro1.txt','r',newline='') as f:\n",
+    "    obj=f.readlines()[1::2]\n",
```

Comment:
any other way to print only even rows?

Reply:
if obj % 2 == 0:
        print(obj, end=" ")

Reply:
good..!!!